Services/Certificate/classes/Placeholder/Description/class.ilStudyProgrammePlaceholderDescription.php

```diff
@@ -1,32 +1,17 @@
-<?php
+<?php declare(strict_types=1);
+
 /* Copyright (c) 1998-2018 ILIAS open source, Extended GPL, see docs/LICENSE */
 
 class ilStudyProgrammePlaceholderDescription implements ilCertificatePlaceholderDescription
 {
-    /**
-     * @var ilDefaultPlaceholderDescription
-     */
-    private $defaultPlaceHolderDescriptionObject;
+    private ?ilDefaultPlaceholderDescription $defaultPlaceHolderDescriptionObject;
+    private ?ilLanguage $language;
```

Comment:
These types are not optional. Only the passed constructor arguments are optional.